src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
+      action="#"
+      method="post"
+      data-qa="big"
+      name="bigForm"
+    >
+      <lebel class="searchBar">
+        <img
+          src="images/Search.svg"
+          alt="Search image"
+          class="searchBar__img searchBar__img--big"
+        >
+        <input type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="searchBar__input searchBar__input--big"
+          data-qa="keypress"
+        >
+      </lebel>
+    </form>
+
+    <form class="form"
+      action="#"
+      method="post"
+      data-qa="small"
+      name="smollForm"
+    >
```

Comment:
```suggestion
<form 
  class="form"
  action="#"
  method="post"
  data-qa="small"
  name="smollForm"
>
```
Fix indentations accordingly to the code style guide kit for all tags in the code